modules/ndu/src/backend/conditions.ts

```diff
@@ -122,5 +122,29 @@ export const dialogConditions: sdk.Condition[] = [
     evaluate: () => {
       return 1
     }
+  },
+  {
+    id: 'type_text',
+    label: 'The user typed something specific',
+    description: `The user typed {text}`,
+    params: {
+      text: { label: 'One or multiple words to detect (one per line)', type: 'array', rows: 5 },
```

Comment:
can you rename text for match or candidate or options. anything more specific then text would work and make it less confusing in the matching function